src/pages/keyword-generator/KeywordGenerator.vue

```diff
@@ -18,62 +15,45 @@ const generate = (keywords, n) => {
   return Array.from(result);
 }
 
-const run = () => {
-  if (text.value.trim() === '') {
-    showWords.value = false;
-    keywords.value = [];
-  } else {
-    onegrams.value = generate(text.value, 1);
-    twograms.value = generate(text.value, 2);
-    threegrams.value = generate(text.value, 3);
-    keywords.value = [...onegrams.value, ...twograms.value, ...threegrams.value];
-    showWords.value = true;
+const generateNGrams = () => {
+  keywords.value = [];
+  if (text.value.trim() !== '') {
+    for(let i = 1;i <= maxNGrams.value; i++) {
+      keywords.value.push({i,keywords: generateWordArray(text.value, i)});
+    }
   }
```

Comment:
Here the code only continues if the condition is met. So we can use early return if the condition is not met.